bemuse/src/game/input/touch-plugin.js

```diff
@@ -62,9 +62,7 @@ export function TouchPlugin(context) {
       output['p1_pinch'] = getPinch(input)
       return output
     },
-    destroy() {
-      statsRecorder.done()
-    },
+    destroy() {},
```

Comment:
`statsRecorder` is not a Promise, have to bring this back.

Reply:
Sorry. Fixed on `7f9a39ef`.